utils.sh

```diff
@@ -26,14 +26,15 @@ function die() {
 
 # Expands '-k' to an absolute path to a kernel image if necessary
 function get_full_kernel_path() {
-    # If '-k' is an path that ends in the kernel image, we can just use it directly
-    if [[ ${KERNEL_LOCATION##*/} = "${KIMAGE:=zImage}" ]]; then
+    # If '-k' is a file, we can just use it dierctly
```

Comment:
s/dierctly/directly/

Reply:
I will use `set spell`, I will use `set spell`, I will use `set spell`...

Fixed, thanks!

Reply:
if only `:set spell` could apply only to comments...